posts/Davide/2017-10-16-hibernate-ogm-5-2-Beta1-released.adoc

```diff
@@ -0,0 +1,78 @@
+= "Hibernate OGM 5.2 Beta1 is out"
+Davide D'Alto
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+Another month, another release: http://hibernate.org/ogm/[Hibernate OGM] 5.2 Beta1 is ready!
+
+The main goal for this release is to support Infinispan 9.1 and allow the creations of caches
+when they are missing.
+
+If you need to upgrade from a version before 5.2, you can find help on the
+https://developer.jboss.org/wiki/HibernateOGMMigrationNotes[migration notes].
+
+All the changes are described in the
+https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10160&version=31400[release notes].
+
+== Infinispan remote automatic cache creation
+
+By default, Hibernate OGM expects the caches you defined via the mapping and the configuration
+to already exit on the Infinispan server. At start up, an exception is thrown if one of the caches
+is not found.
+ 
+With this version is now possible to have Hibernate OGM create the caches on startup if they are
+missing. You can enable this feature setting the property:
+
+```
+hibernate.ogm.create_database=true
+```
+
+The official documentation has more details about the
+https://docs.jboss.org/hibernate/ogm/5.2/reference/en-US/html_single/#_cache_names_used_by_hibernate_ogm[different cache mapping approaches]
+in Infinispan.
+
+[NOTE]
+====
+We understand that the name of this property is strange in the Infinispan context.
+This property is a generic one used by many different dialects and it will likely change before
+the final release.
+====
+
+== Where can I get it?
+
+You can get the required core library using the following Maven coordinates:
+
+* _org.hibernate.ogm:hibernate-ogm-core:5.2.0.Beta1_ 
```

Comment:
Since the core artifact is pulled as a transitive dependency by the backends, not sure it's worth mentioning?
We don't mention it on [hibernate.org](http://hibernate.org/ogm/releases/5.2/#get-it).

By the way, to avoid that kind of issues, in my [last release post for Hibernate Search](http://in.relation.to/2017/10/04/hibernate-search-5-8-1-Final/) I simply linked to the proper page on hibernate.org.

Reply:
I prefer if they don't have to click an additional link to get the coordinates for the released version.
I will add the link to the download page, though.

OK, I'll remove the core coordinates.